<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="aswin779" failures="1" tests="1" name="Test-Runner 1" time="37.594" errors="0" timestamp="2023-02-08T10:41:08 IST">
  <testcase classname="separatepackage.CreatecontactWithBase" name="createcontactpomtest" time="22.383">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at separatepackage.CreatecontactWithBase.createcontactpomtest(CreatecontactWithBase.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1589)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- createcontactpomtest -->
</testsuite> <!-- Test-Runner 1 -->
